dds/DCPS/DataReaderImpl.cpp

```diff
@@ -3282,6 +3282,56 @@ DataReaderImpl::get_ice_endpoint()
   return TransportClient::get_ice_endpoint();
 }
 
+DDS::ReturnCode_t DataReaderImpl::setup_deserialization()
+{
+  const DDS::DataRepresentationIdSeq repIds =
+    get_effective_data_rep_qos(qos_.representation.value);
+  bool success = false;
+  if (cdr_encapsulation()) {
+    Encoding::Kind encoding_kind;
+    for (CORBA::ULong i = 0; i < repIds.length(); ++i) {
+      if (repr_to_encoding_kind(repIds[i], encoding_kind)) {
+        if (Encoding::KIND_XCDR1 == encoding_kind ||
+            Encoding::KIND_XCDR2 == encoding_kind) {
+          if (decoding_modes_.find(encoding_kind) == decoding_modes_.end()) {
+            Encoding encoding(encoding_kind, swap_bytes());
+            decoding_modes_.insert(std::make_pair(encoding_kind, encoding));
+            success = true;
+          }
+        } else if (::OpenDDS::DCPS::DCPS_debug_level >= 2) {
```

Comment:
Since this code isn't in a template anymore, there's no reason for the full name. Same for the other log level references.